Automate/RedHatConsulting_LDAP/Integration/LDAP/StateMachines/Methods.class/__methods__/execute_automation_request_against_all_vms_in_batches.rb

```diff
@@ -118,7 +118,6 @@ def execute_automation_request_for_batch_of_vms(vm_ids, automation_request_insta
   # update the VMs in asyncronous batches
   $evm.log(:info, "VMs to Update in Batches: { vms => #{vms.collect { |vm| vm.name }} }") if @DEBUG
   vms.each_slice(vms_batch_size) do |vms_batch|+  error("automation_request_instance_name msut be specified") if automation_request_instance_name.blank?
-￼
```

Comment:
There was a weird unicode character here that doesn't show up in the Github web interface